users/users.py

```diff
@@ -183,6 +185,9 @@ def read_all_users_ldap(filters=[], include_temp=False):
 
     @staticmethod
     def get_balance(user_id, session=get_session()):
+        if not is_pi():
+            return -50
```

Comment:
why?

Reply:
Because I did not have the DB structure information ;)